### ISSUE TITLE
The tab(s) still active while exit browser

### ISSUE THREAD

**Issue by gracie95** [Slion/Fulguris#447]
# Description
Cache, etc. cleared after exiting, but the not the remain tab(s).

# Environment
 - Device: Galaxy J6 (2018)
 - OS: Android 10
 - Version: 1.8.18

# To reproduce
Steps to reproduce the behaviour:
1. Make sure **Exit menu item** has active.
2. Open one or more tabs (recommended: contain loaded pages)
3. Go to browser options and press **Exit**.
4. Open again the browser. The remain tab(s) will be reloaded.

# Expected behaviour
The tab should be empty while reopen app.

**Comment by Slion** [comment]
You would like all your tabs to close when you exit the app?

**Comment by Slion** [comment]
Yeah that's not working well with Fulguris unless you use private/incognito mode.

**Comment by The-First-King** [comment]
As a workaround only the bookmarks tab can be left opened before exit from the web browser.

**Comment by Cwpute** [comment]
> As a workaround only the bookmarks tab can be left opened before exit from the web browser.

Does that make the previously open tabs close when you open Fulguris again ? @The-First-King

**Comment by The-First-King** [comment]
No it's not. As I mentioned, it's necessary to close all tabs manually and open bookmarks tab before exit the web browser.

If some tabs are still needed then they can be saved in sessions.

**Comment by Lavmint** [comment]
> You would like all your tabs to close when you exit the app?

When user closes the app it shall work the same way as in other browsers: no trace of the last opened tabs unless they were pinned. Nether in the Settings->Domains, nether while opening browser. Home page shall be opened as on an average start of the app without any tabs opened.

P.S. I personally have my desktop browsers configured to keep only 1 cookie, because I don't like to authorize myself in google as it's time consuming
